platform/darwin/src/MGLCircleStyleLayer.h

```diff
@@ -17,6 +17,8 @@ typedef NS_ENUM(NSUInteger, MGLCircleStyleLayerCirclePitchScale) {
 
 @interface MGLCircleStyleLayer : MGLBaseStyleLayer <MGLStyleLayer>
 
+@property (nonatomic) NSPredicate *predicate;
```

Comment:
#6076 has landed, so without a nullability qualifier, the compiler will assume that the predicate is non-`nil`.